mubench.pipeline/requirements.py

```diff
@@ -93,31 +91,43 @@ def _get_cpu_count(self):
         if _in_container():
             return self._get_container_cpu_count()
         else:
-            return psutil.cpu_count(logical=False)
+            return self._get_normal_cpu_count()
 
     def _get_container_cpu_count(self):
         try:
             with open("/sys/fs/cgroup/cpu/cpu.cfs_quota_us") as file:
-                return int(file.readline()) / 100000
+                quota = int(file.readline())
+
+            no_limit = quota < 0
+            if no_limit:
+                return self._get_normal_cpu_count()
+            else:
+                return quota / 100000
```

Comment:
Could you add a comment as to why you divide by 100000? This is not obvious (to me).